app/src/pages/inside/logsPage/historyLine/historyLine.jsx

```diff
@@ -72,36 +76,41 @@ export class HistoryLine extends Component {
     this.state = {
       isLoading: false,
       prevItemsCount: DEFAULT_HISTORY_DEPTH,
-      shouldShowLoadMore: false,
     };
   }
   static propTypes = {
     intl: PropTypes.object.isRequired,
     historyItems: PropTypes.array,
     activeItemId: PropTypes.number,
+    shouldShowLoadMore: PropTypes.bool,
     changeActiveItem: PropTypes.func,
     fetchHistoryItemsAction: PropTypes.func,
+    setShouldShowLoadMoreAction: PropTypes.func,
   };
 
   static defaultProps = {
     historyItems: [],
     activeItemId: 0,
+    shouldShowLoadMore: false,
     changeActiveItem: () => {},
     fetchHistoryItemsAction: () => {},
+    setShouldShowLoadMoreAction: () => {},
   };
 
   checkIfTheItemLinkIsActive = (item) =>
     item.id !== this.props.activeItemId && item.status !== NOT_FOUND;
 
   finishLoading = () => {
+    this.setState({ isLoading: false });
```

Comment:
Why we need another one `setState` here? It is already exists on line [112](https://github.com/reportportal/service-ui/pull/2616/files#diff-201de2248fe02bf80d812a212f8bfa94449055e03ec1fbe3435ad9a986dbfdb2R112)